js/clinicalTimeline.js

```diff
@@ -782,29 +819,23 @@ window.clinicalTimeline = (function(){
       var dayFormat = [];
       var m;
       var d;
-      if (time.y !== 0) {
-        dayFormat = dayFormat.concat(time.y+"y");
-      }
-      if (time.m !== 0) {
-        if (time.y !== 0) {
-          m = Math.abs(time.m);
-        } else {
-          m = time.m;
-        }
-        dayFormat = dayFormat.concat(m+"m");
-      }
-      if (time.d !== 0) {
-        if (time.y !== 0 || time.m !== 0) {
-          d = Math.abs(time.d);
-        } else {
-          d = time.d;
-        }
-        dayFormat = dayFormat.concat(d+"d");
-      }
       if (time.y === 0 && time.m === 0 && time.d === 0) {
         dayFormat = [0];
```

Comment:
This doesn't need to be an array anymore, since we are not using `dayFormat.join("")` any longer. For clarity change to 0.